docs/operations.rst

```diff
@@ -9,7 +9,7 @@ Operators
 ---------
 
 :obj:`COO` objects support a number of operations. They interact with scalars,
-:obj:`numpy.ndarray` objects, other :obj:`COO` objects, and
+:doc:`Numpy arrays <reference/generated/numpy.ndarray>`, other :obj:`COO` objects, and
```

Comment:
I'm curious, why was this change required?

Reply:
Saying "numpy dot enn dee array objects" is a mouthful. I thought it would be easier for readers to just change it to "Numpy arrays."